src/httpp/http/Parser.cpp

```diff
@@ -101,6 +101,30 @@ static bool parse_method(Iterator& it, Request& request)
                     return true;
                 }
             }
+            else if (*it == 'R')
```

Comment:
since the stream based parser is deprecated and you'll add many more methods, maybe we can either delete it or simplify it ?